plugins/shared/loader/loader_test.go

```diff
@@ -55,8 +56,12 @@ func newHarness(t *testing.T, plugins []string) *harness {
 		t.Fatalf("failed to get self executable path: %v", err)
 	}
 
+	exeSuffix := ""
+	if runtime.GOOS == "windows" {
+		exeSuffix = ".exe"
+	}
 	for _, p := range plugins {
-		dest := filepath.Join(h.tmpDir, p)
+		dest := strings.Join([]string{filepath.Join(h.tmpDir, p), exeSuffix}, "")
```

Comment:
Does the following work?

```suggestion
		dest := filepath.Join(h.tmpDir, p) + exeSuffix
```